packages/schema-type/src/typescript/schemaAbiTypes.ts

```diff
@@ -1,5 +1,5 @@
 import { AbiType } from "abitype";
-import { TupleSplit } from "./utils";
+import { TupleSplit } from "../../../protocol-parser/src/utils";
```

Comment:
seems like we need to move TupleSplit here as well to avoid a circular dependency?

Reply:
whoops, yep! thanks